components/director/internal/domain/formation/service.go

```diff
@@ -1161,3 +1201,58 @@ func setToSlice(set map[string]bool) []string {
 	}
 	return result
 }
+
+func (s *service) processFormationNotifications(ctx context.Context, formation *model.Formation, formationReq *webhookclient.FormationNotificationRequest, state model.FormationState) error {
```

Comment:
`errorState` seems more suitable because in both cases we pass "error states", either "create error state" or "delete error state". But I'm not sure the `state` is confusing because both `CreateErrorFormationState` and `DeleteErrorFormationState` are types of `state`, right? `errorState` is just a subset of a `state` options

Reply:
At least initially I interpreted it to be the desired or final state after successfully processing the notification, but it isn't a big deal.